static/css/main.css

```diff
@@ -0,0 +1,20 @@
+.student {
+    padding-bottom: 20px;
+}
+
+.students {
+    list-style: none;
+}
+nav {
+    background-color: #38474e !important;
+}
+
+.page-footer {
+    background-color: #37474f !important;
+}
+
+canvas {
```

Comment:
Better to give this an id, and use that in the pages.

Reply:
@ashwini0529 maybe this got missed?

Reply:
Hmm, this moved from `activity.html` , probably because the content was being rendered on the main page also.  It definitely needs an id if it is being moved here, otherwise it may play badly with other use of `canvas` in another (future) app.

but IMO, this patch doesnt need to improve `activity.html`.  It makes this patch to 'busy'.